README.md

```diff
@@ -155,3 +155,15 @@ The path will be added to the mnemonic phrase of the parent account.
 * `Module not found: Error: Can't resolve './chains-metadata.json'`
   
   The Signer bundles in the most recent (for the time of building) metadata for the production and testnet Aleph Zero chains. Apparently, the json file containing them is absent, so make sure to generate it by executing `yarn download-chains-metadata` before bundling the application.
+
+
+## Signer icons
+
+Feel free to use those icon when integrating your app with Signer
```

Comment:
Lovely!

Should we add an svg as well? I think I used mostly svgs when updating our dapps.

Reply:
those are the icons I got from @bryk, if you think the svg is important I can add it as well, but to me it should suffice :shrug:

Reply:
I'll add it to short-circuit the issue :+1: :sweat_smile: